### PR TITLE
Adding jaejer of file otel local examples

### PR DESCRIPTION
**Pull request by yzcyayaya**
**Description:** <Describe what has changed. 
Ex. Fixing a bug - Describe the bug and how this fixes the issue.
I'm having a configuration  problem, and maybe others are having a configuration problem
Ex. Adding a feature - Explain what this achieves.>
Adding jaejer of file otel local examples

**Link to tracking Issue:** <Issue number if applicable>
[https://github.com/open-telemetry/opentelemetry-collector/issues/5845](issues url)

### CODE REVIEW

File: examples/local/otel-config.yaml [hunk 1]
```diff
@@ -8,6 +8,7 @@ receivers:
   otlp:
     protocols:
       grpc:
+        endpoint: 0.0.0.0:4317
```
Comment (jpkrohling) [inline examples/local/otel-config.yaml:11]: Why do you need this?

Comment (yzcyayaya) [reply examples/local/otel-config.yaml]: I have deleted him in the new commit

Comment (yzcyayaya) [reply examples/local/otel-config.yaml]: > Why do you need this?

Excuse me, Can this PR be merged?

File: examples/local/otel-config.yaml [hunk 2]
```diff
@@ -22,13 +22,15 @@ processors:
 exporters:
   logging:
     loglevel: debug
+  jaeger:
```
Comment (jpkrohling) [inline examples/local/otel-config.yaml:25]: We don't have any other backends in this example, why should we have Jaeger?

**Comment by jpkrohling** [comment]
I'm closing this due to a lack of feedback from the author.

**Pull request closed without merging**